### ISSUE TITLE
 E0611 when .pyd file with same name as file exists

### ISSUE THREAD

**Issue by SuperStormer** [PyCQA/pylint#2468]
### Steps to reproduce
1. Create a file `a.py` 
`class A:
pass`
2.  Create a file `b.py` 
`from a import A`
3. Create a file `a.pyd`, leave it blank

### Current behavior
>************* Module b
b.py:1:0: C0304: Final newline missing (missing-final-newline)
b.py:1:0: C0111: Missing module docstring (missing-docstring)
b.py:1:0: E0611: No name 'A' in module 'a' (no-name-in-module)
b.py:1:0: W0611: Unused A imported from a (unused-import)

-----------------------------------------------------------------------
>Your code has been rated at -70.00/10 (previous run: -50.00/10, -20.00)

### Expected behavior
Every line but `b.py:1:0: E0611: No name 'A' in module 'a' (no-name-in-module)` should be printed

### pylint --version output
pylint 2.1.1
astroid 2.0.4
Python 3.6.6 |Anaconda custom (64-bit)| (default, Jun 28 2018, 11:27:44) [MSC v.1900 64 bit (AMD64)]

**Comment by mosrosem** [comment]
Not happened to me, I installed:
pylint 2.1.1
astroid 2.0.4
Python 3.6.6
and I got this:
```bash
************* Module b
b.py:1:0: C0111: Missing module docstring (missing-docstring)
b.py:1:0: W0611: Unused A imported from a (unused-import)

-----------------------------------------------------------------------
Your code has been rated at -10.00/10 (previous run: -20.00/10, +10.00)
```

**Comment by PCManticore** [comment]
Hi folks. I also cannot reproduce this. @SuperStormer maybe there is something missing in your reproduction example?
